utils/types.ts

```diff
@@ -1 +1 @@
-export type IconKey = 'GITHUB' | 'LINKEDIN' | 'TWITTER'
\ No newline at end of file
+export type IconKey = 'GITHUB' | 'LINKEDIN' | 'TWITTER' |'MENU'|'CLOSE'
```

Comment:
```suggestion
export type IconKey = 'GITHUB' | 'LINKEDIN' | 'TWITTER' | 'MENU' | 'CLOSE'
```
Add blank spaces between options